sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -211,7 +211,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
         tryDownloadSpark(version, sparkTestingDir.getCanonicalPath)
       }
 
-      // Extract major.minor for testing Spark 3.1.x and 3.0.x with metastore 2.3.9 and Java 11.
+      // Extract major.minor for testing Spark 3.1.x and 3.0.x with metastore 2.3.10 and Java 11.
```

Comment:
Instead of updating this, it seems that we had better remove this comment because this is very outdated in many ways.
- We don't testing Spark 3.1.x and 3.0.x
- We don't test Java 11

Reply:
Or, simply let's revert from this PR to reduce diff size.

Reply:
removed